admin/autoupdater/files/usr/sbin/autoupdater

```diff
@@ -41,6 +43,9 @@ local function parse_args()
   while arg[i] do
     if arg[i] == '-f' then
       force = true
+    elseif arg[i] == '-F' then
```

Comment:
As mentioned in the other bug report, I'd prefer a more verbose flag here, something like `--force-version` or `--ignore-version`.